release_notes/versioning-policy.adoc

```diff
@@ -1,11 +0,0 @@
-[id="ocp-4-2-versioning-policy"]
-= {product-title} Versioning Policy
-include::modules/common-attributes.adoc[]
-:context: release-notes
-
-Do not add or edit release notes here. Edit release notes directly in the branch
-that they are relevant for.
-
-Release note changes should be added/edited in their own PR.
```

Comment:
With 4.9's #39184, a good chunk of the `oc` skew docs are moving over to `cli_reference/openshift_cli/usage-oc-kubectl.adoc`.  Now that they're outside of `release_notes`, can those get picked up in `main` in this PR, so that they will no longer need to be grafted onto each release branch after it forks off?